frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -346,7 +278,7 @@ export default {
     },
     detect_is_ok_to_save: async function () {
       if (this.has_changed) {
-        await this.save();
+        this.$emit('save');
```

Comment:
Propagating `save` event to the parent so it can call saving logic for annotations